README.md

```diff
@@ -83,3 +83,33 @@ results of Docker image upgrades.
 See the [ECR Sandbox Readme](ecr-sandbox) for setup instructions.
 
 [ecr-sandbox]: terraform/ecr-sandbox/README.md
+
+### Generate Release Versions
+
+This is a python script which is run on Jenkins. It generates an html file and a Slack message which show which repositories have out of date versions deployed to the staging and production environments. It does this by looking at the release tags on each of the `release-$environment` branches.
+
+#### Running on Jenkins
+The job to run the script is [here](https://jenkins.tdr-management.nationalarchives.gov.uk/job/Github%20release%20summary/) and when the job is run, it generates [an html file](https://jenkins.tdr-management.nationalarchives.gov.uk/job/Github%20release%20summary/7/Release_20Version_20Report/) like this one. 
+
+#### Running locally
+There is only one dependency outside the standard python library which will need to be installed to run this locally. You can install this system wide by running `pip install quik` but it's better to use a virtual environment.
```

Comment:
Any reason not to use requirements.txt? It seems to be pretty standard in Python.

Reply:
No reason really. It didn't seem worth it for one dependency but as you say, it's the standard way to do it so I've put one in.